pals/mysterious-halfling.md

```diff
@@ -0,0 +1,65 @@
+
+# ??? It Is A Mystery
```

Comment:
You should come up with a name; I've been using heptambic nearly-alliterative for halfing NPCs (eg Grumblefront Goldenpalm, Riddley Rumblefruit) if you'd like to fit in, but it's fine to be different (halflings come from lots of different countries & cultures, after all)